package.json

```diff
@@ -26,16 +26,16 @@
     "types": "tsc -b ."
   },
   "resolutions": {
-    "@babel/parser": "7.16.0",
-    "@babel/traverse": "7.16.0",
-    "@babel/types": "7.16.0",
+    "@babel/parser": "7.19.0",
+    "@babel/traverse": "7.19.0",
+    "@babel/types": "7.19.0",
```

Comment:
Initially I couldn't find the `DeclareFunction` node in the AST so I upgraded babel, but wasn't the issue.  I think it's fine to keep this upgrade though.

Reply:
This means that they now differ from webapp, not sure if that matters or not?